modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -2,11 +2,15 @@ add-new-related-queries-based-on-successful-queries=Add New Related Queries Base
 add-x-as-a-search-provider=Add {0} as a Search Provider
 aggregation-field=Aggregation Field
 aggregation-field-help=The name of the indexed field to aggregate results by. Must be a non-analyzed keyword field.
+allow-empty-searches=Allow Empty Searches
+allow-empty-searches-help=When enabled, an empty search query returns all available assets. By default, an empty query returns no search results.
 any-asset=Any Asset
 any-category=Any Category
 any-folder=Any Folder
 any-site=Any Site
 any-user=Any User
+basic-facet-selection=Basic Facet Selection
+basic-facet-selection-help=When enabled, facet selections still filter search results, but facet counts remain unaffected.
```

Comment:
Is this really just about facet counts, or is there something else to it? If it's all about counts, we might consider renaming the property to be more clear?

>basic-facet-selection-help=When enabled, facet counts are not recalculated on facet selection.

Reply:
The other visible effect (harder to explain without a screenshot) is that the lists of grouped terms themselves remain unchanged. In "Basic" mode, you always see all groups (and their counts) for the given keywords, even elements that may not match visible search results after narrowing it down with a few selections. In "Smart" mode (the default now) the grouped terms themselves are also narrowed down to match the search results you see after selections, and some terms may even disappear from lists if they don't correspond to any visible results.

Open to suggestions 😃

Reply:
Then the above suggestion is fine. The user should understand that no recounting happens, so the facet availability UI isn't updated. I'll offer a couple alternatives below, but if you like the first suggestion, feel free to use it. It's the simplest.

_Basic Facet Selection_ is fine for this purpose. The only possible alternative would be swapping _Static_ for _Basic_, but I'm not sure which is better here. You can decide which you like.
A couple possible alternatives for the help text:

>basic-facet-selection-help=When enabled, available facets and facet counts are calculated once. Facet selection does not result in any facet availability update. 

>basic-facet-selection-help=When enabled, facet applicability and counts are not recalculated following facet selection.